### DISCUSSION TITLE
Unable to download opensearch-cluster helm chart from remote repository

### DISCUSSION THREAD

**Discussion by xkovacsp1** [Opster/opensearch-k8s-operator#563] [Q&A] [answered]
Hello,
currently, I am trying to install opensearch-cluster according to this guide: https://github.com/Opster/opensearch-k8s-operator/blob/main/docs/userguide/cluster-chart.md#install-chart .
But unfortunately, I get this error:

![image](https://github.com/Opster/opensearch-k8s-operator/assets/22496898/f17a251f-7cc0-485a-a869-e4240a9e23b8)

As you can see the opensearch-operator repository is added to my repositories

![image](https://github.com/Opster/opensearch-k8s-operator/assets/22496898/971162da-cb40-42d3-9c24-c616d25c4ec3)

Thanks for your help.

**Comment by swoehrl-mw** [comment, marked as answer]
Hi @xkovacsp1. The cluster chart is not yet part of a release. The docs you are using represent the current development version. If you want to test the chart now you will need to checkout the repo and use the helm chart from there (`helm install test ./opensearch-k8s-operator/charts/opensearch-cluster`).
